reference/docs-conceptual/whats-new/breaking-changes-ps6.md

```diff
@@ -64,6 +64,16 @@ functionality with new functionality and a redesigned syntax:
 Due to the use of unsupported APIs, `Microsoft.PowerShell.LocalAccounts` has been removed from
 PowerShell Core until a better solution is found.
 
+### `*-Computer` cmdlets
+
+Due to the use of unsupported APIs, the following cmdlets been removed from PowerShell Core until a
```

Comment:
"cmdlets **have** been removed"

Reply:
Fixed. And, yes, this is targeted at staging. These were removed in the PS6.0 release.